modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/skin/Utils.java

```diff
@@ -213,8 +212,29 @@ public static int computeTruncationIndex(Font font, String text, double width) {
         return index;
     }
 
-    public static String computeClippedText(Font font, String text, double width,
-                                     OverrunStyle type, String ellipsisString) {
+    /**
+     * Computes the actual text to be shown in the Labeled with the text wrapping disabled:
+     * unmodified if it fits into available area,
+     * or with the ellipsis string inserted into strategic place(s) if it does not.
```

Comment:
ellipsis string can be referenced similar to textTruncated?

Reply:
Can you please clarify your question?  I don't understand.

Reply:
While that would be an OK change, this is an internal method, so it really doesn't much matter -- the docs are only seen by those looking at the code.